config/data/bsnl_template_details.yml

```diff
@@ -0,0 +1,262 @@
+# This is an autogenerated file. Do not modify.
+# Use get_bsnl_template_details.rake to fetch a new copy of this file.
+---
+hi-IN.notifications.set01.basic:
+  Template_Id: '1407164786628639161'
+  Template_Keys: []
```

Comment:
Is this because the vars haven't been named?

Reply:
Yes. These should look like the `en` messages below when we fetch this list after naming.